### DISCUSSION TITLE
How to correctly configure log from another container? (Kubernes)

### DISCUSSION THREAD

**Discussion by guilhermemauro** [apache/airflow#26160] [Q&A]
Hello guys, my airflow web if point to name server that doesn't exists, this name server is actually, the name of my pod. How can I configure it to point to the correct service name?

![image](https://user-images.githubusercontent.com/12633498/188430667-737fd65f-7da8-40ea-ad4b-e1c0f2d030a3.png)

**Comment by potiuk** [comment]
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html?highlight=callable#hostname-callable

**Reply by guilhermemauro** [reply]
yeah, I saw it. But I just wanna to pass a simple string and it needs to be a callable.

**Reply by potiuk** [reply]
Why not callable returning a string ?

**Reply by guilhermemauro** [reply]
But... How? I got confused. Can I pass reference for custom python file or just python modules installed by pip?

**Reply by potiuk** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/faq.html?highlight=airflow_local_settings#airflow-local-settings-py - people usually use airflow_local_settings.py which precisely foreseen for that purpose

**Reply by potiuk** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/concepts/cluster-policies.html?highlight=airflow_local_settings
